pkg/dashboard/router.go

```diff
@@ -69,20 +72,23 @@ func GetRouter(setters ...Option) *mux.Router {
 	router.Handle("/namespaces", NamespaceList(*opts))
 
 	// root
-	router.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-		// catch all other paths that weren't matched
-		if r.URL.Path != "/" && r.URL.Path != opts.BasePath && r.URL.Path != opts.BasePath+"/" {
-			klog.Infof("404: %s", r.URL.Path)
-			http.NotFound(w, r)
-			return
-		}
-
-		klog.Infof("redirecting to %v", path.Join(opts.BasePath, "/namespaces"))
-		// default redirect on root path
-		http.Redirect(w, r, path.Join(opts.BasePath, "/namespaces"), http.StatusMovedPermanently)
-	})
+	router.HandleFunc("", rootHandler)
```

Comment:
Is this actually necessary? I think I've always just used the "/" handler.

Reply:
Unfortunately if I don't add this new empty handler then from what I've seen a call to `example.com/foo` wouldn't redirect to the namespaces page, whilst a forward slash at the end would work. I added this so that the custom basePath would be inline with how the default works, as the default option redirects flawlessly without any issue with or without a forward slash at the end of the URL

Reply:
Weird. My gut says there's a better way to do all of this, but if it's working, then I'm good with it.

Reply:
I used mux's StrictSlash() in a previous commit but couldn't get it working. However, looking it up just now has made me realise that I used it wrong, and it seems that when using it the extra handler is unnecessary 😅
I'm implementing and testing this now, I should be able to get a commit out shortly to address this!

Reply:
I've simplified router.go in b849cd7 by reverting it mostly back to how it was